internal/ark/sync/restores_sync_svc.go

```diff
@@ -60,9 +61,21 @@ func (s *RestoresSyncService) SyncRestores(clusterManager *cluster.Manager) erro
 	}
 
 	for _, cluster := range clusters {
-		err := s.syncRestoresForCluster(cluster)
+		log := s.logger.WithField("clusterID", cluster.GetID())
+
+		status, err := cluster.GetStatus()
+		if err != nil {
+			log.Error(emperror.Wrap(err, "could not get cluster status"))
```

Comment:
Use an error handler here.

Reply:
Thanks @sagikazarmark , I'll fix this in an upcoming PR!